Sources/web3swift/Web3/Web3.swift

```diff
@@ -12,9 +12,9 @@ extension Web3 {
 
     /// Initialized provider-bound Web3 instance using a provider's URL. Under the hood it performs a synchronous call to get
     /// the Network ID for EIP155 purposes
-    public static func new(_ providerURL: URL) async throws -> Web3 {
+    public static func new(_ providerURL: URL, network net: Networks = .Mainnet) async throws -> Web3 {
```

Comment:
minor: but why not just call it `network`?

Reply:
I copied it from Web3HttpProvider